changelog/16.0/16.0.2/summary.md

```diff
@@ -0,0 +1,7 @@
+## Summary
+
+### Shard name validation in TopoServer
+
+Prior to v16.0.1, it was possible to create a shard name with invalid characters, which would then be inaccessible to various cluster management operations.
```

Comment:
Should this be 16.0.2?

Reply:
ugh, i was so careful to change it from 17!!!